tests/query/parser/test_conditions.py

```diff
@@ -0,0 +1,338 @@
+                            (Column(None, "b", None), Literal(None, 2)),
+                        ),
+                    ),
+                ),
+                FunctionCall(
+                    None,
+                    ConditionFunctions.EQ,
+                    (Column(None, "c", None), Literal(None, 3)),
+                ),
+            ),
+        ),
+    ),
+    (
+        [[["a", "=", 1], ["b", "=", 2]], [["c", "=", 3], ["d", "=", 4]]],
+        FunctionCall(
+            None,
+            BooleanFunctions.AND,
+            (
+                FunctionCall(
+                    None,
+                    BooleanFunctions.OR,
+                    (
+                        FunctionCall(
+                            None,
+                            ConditionFunctions.EQ,
+                            (Column(None, "a", None), Literal(None, 1)),
+                        ),
+                        FunctionCall(
+                            None,
+                            ConditionFunctions.EQ,
+                            (Column(None, "b", None), Literal(None, 2)),
+                        ),
+                    ),
+                ),
+                FunctionCall(
+                    None,
+                    BooleanFunctions.OR,
+                    (
+                        FunctionCall(
+                            None,
+                            ConditionFunctions.EQ,
+                            (Column(None, "c", None), Literal(None, 3)),
+                        ),
+                        FunctionCall(
+                            None,
+                            ConditionFunctions.EQ,
+                            (Column(None, "d", None), Literal(None, 4)),
+                        ),
+                    ),
+                ),
+            ),
+        ),
+    ),
+    (
+        [[["a", "=", 1], []]],
+        FunctionCall(
+            None, ConditionFunctions.EQ, (Column(None, "a", None), Literal(None, 1)),
+        ),
+    ),  # Malformed Condition Input
+    (
+        [[[["tag", ["foo"]], "=", 1], ["b", "=", 2]]],
+        FunctionCall(
+            None,
+            BooleanFunctions.OR,
+            (
+                FunctionCall(
+                    None,
+                    ConditionFunctions.EQ,
+                    (
+                        FunctionCall(None, "tag", (Column(None, "foo", None),)),
+                        Literal(None, 1),
+                    ),
+                ),
+                FunctionCall(
+                    None,
+                    ConditionFunctions.EQ,
+                    (Column(None, "b", None), Literal(None, 2)),
+                ),
+            ),
+        ),
+    ),  # Test functions in conditions
+    (
+        [["primary_hash", "LIKE", "%foo%"]],
+        FunctionCall(
+            None,
+            ConditionFunctions.LIKE,
+            (Column(None, "primary_hash", None), Literal(None, "%foo%")),
+        ),
+    ),  # Test special output format of LIKE
```

Comment:
I think this was `special` because of the escaping maybe ? 
I copied from here
https://github.com/getsentry/snuba/blob/master/tests/test_util.py#L129-L134

Does not seem very special to me either.